include/swift/AST/DiagnosticsParse.def

```diff
@@ -653,6 +653,13 @@ ERROR(expected_rangle_protocol,PointsToFirstBadToken,
       "expected '>' to complete protocol composition type", ())
 ERROR(disallowed_protocol_composition,PointsToFirstBadToken,
       "protocol composition is neither allowed nor needed here", ())
+ERROR(any_composition_member,PointsToFirstBadToken,
+      "'Any' cannot be a member of a protocol composition", ())
+
+WARNING(deprecated_protocol_composition,PointsToFirstBadToken,
+        "'protocol<...>' composition syntax is deprecated; use infix '&' instead", ())
```

Comment:
I think what @karwa means, isn't about error vs warning, but to use a message better understood by the majority of people.
My suggestion is `"'protocol<...>' composition syntax is deprecated; join the protocols using '&'"`